### PR TITLE
WebSpace Blog

### PR DESCRIPTION
**Pull request by Shockcoil131**
Added webspaces blog since a tutorial isnt available outside of hackerclub slides

### CODE REVIEW

File: posts/2024-03-25-getting-started-with-your-redbrick-webspace.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+## Why?
+You must be asking: "Why would I use this **extremely awesome** and **free** service?" Well allow me to clue you in. Redbrick webspaces are extremely easy to setup, with just a few simple commands to get you up and running. They are also a great way to host a variety of different (static) sites.
+
+## How?
+So how do you make use of this great service? Well first you need a site to use. A great first site to use is the portfolio you created for CA106 (Web Design). You could also develop something else, either on your own system or directly on Redbrick server via SSH. For the sake of this tutorial we will assume a locally developed website to be pushed to your webspace.
+
+So first you pick and SFTP client of your choice. Most either gravitate to either [WinSCP](https://winscp.net/eng/index.php) or [FileZilla](https://filezilla-project.org/). I myself prefer WinSCP. The information you need to affectively connect to Redbrick servers with clients is shown bellow:
```
Comment (wizzdom) [inline posts/2024-03-25-getting-started-with-your-redbrick-webspace.md:21]:
```suggestion
So first you pick an SFTP client of your choice. Most either gravitate to either [WinSCP](https://winscp.net/eng/index.php) or [FileZilla](https://filezilla-project.org/). Personally, I prefer WinSCP. The information you need to effectively connect to Redbrick servers with clients is shown below:
```

Comment (DistroByte) [inline posts/2024-03-25-getting-started-with-your-redbrick-webspace.md:21]:
```suggestion
So first you pick and SFTP client of your choice. Most either gravitate to either [WinSCP](https://winscp.net/eng/index.php) or [FileZilla](https://filezilla-project.org/). I myself prefer WinSCP. The information you need to effectively connect to Redbrick servers with clients is shown bellow:
```

File: posts/2024-03-25-getting-started-with-your-redbrick-webspace.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+```
+
+For any other issues you may have, you can message in `#helpdesk` in the discord server.
+
+## Outro
+Hopefully now you have your very own Redbrick webspace that you can show off to your friends and ~~shill Redbrick~~ let them know how than can have their own!
+
+If you have any issues/questions drop a message in `#helpdesk`.
+
+~fbaxter
```
Comment (wizzdom) [inline posts/2024-03-25-getting-started-with-your-redbrick-webspace.md:72]:
```suggestion
`~fbaxter`
```

**Review by wizzdom — COMMENTED** [review]
Looking good! Just a few small suggestions

File: posts/2024-03-25-getting-started-with-your-redbrick-webspace.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+```
+
+Now you can go to `https://your_redbrick_username.redbrick.dcu.ie` and see your amazing website!
+
+## Potential Issues
+The two main issues you can encounter are 403 or 404 errors.
+
+To resolve the 403 error you need to ensure you've run the two chmod commands that were shown above, especially if you've replaced files or added new ones. 
+
+For 404 errors you need to make sure that the site structure follows something similar to bellow. Particularly the placement and naming of `index.html`.
```
Comment (DistroByte) [inline posts/2024-03-25-getting-started-with-your-redbrick-webspace.md:58]:
```suggestion
For 404 errors you need to make sure that the site structure follows something similar to below. Particularly the placement and naming of `index.html`.
```

File: posts/2024-03-25-getting-started-with-your-redbrick-webspace.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+```
+public_html
+	index.html
+	{other_files}
+```
+
+For any other issues you may have, you can message in `#helpdesk` in the discord server.
```
Comment (DistroByte) [inline posts/2024-03-25-getting-started-with-your-redbrick-webspace.md:65]: Can we link to the helpdesk channel here?

File: posts/2024-03-25-getting-started-with-your-redbrick-webspace.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+## Outro
+Hopefully now you have your very own Redbrick webspace that you can show off to your friends and ~~shill Redbrick~~ let them know how than can have their own!
+
+If you have any issues/questions drop a message in `#helpdesk`.
```
Comment (DistroByte) [inline posts/2024-03-25-getting-started-with-your-redbrick-webspace.md:70]: Same as above

File: posts/2024-03-25-getting-started-with-your-redbrick-webspace.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+To resolve the 403 error you need to ensure you've run the two chmod commands that were shown above, especially if you've replaced files or added new ones. 
+
+For 404 errors you need to make sure that the site structure follows something similar to below. Particularly the placement and naming of `index.html`.
+```
+public_html
+	index.html
+	{other_files}
+```
+
+For any other issues you may have, you can message in `#helpdesk` in the discord server.
```
Comment (wizzdom) [inline posts/2024-03-25-getting-started-with-your-redbrick-webspace.md:66]:
```suggestion
For any other issues you may have, you can send a message in [`#helpdesk`](https://discord.redbrick.dcu.ie) in the discord server.
```

File: posts/2024-03-25-getting-started-with-your-redbrick-webspace.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+## Outro
+Hopefully now you have your very own Redbrick webspace that you can show off to your friends and ~~shill Redbrick~~ let them know how than can have their own!
+
+If you have any issues/questions drop a message in `#helpdesk`.
```
Comment (wizzdom) [inline posts/2024-03-25-getting-started-with-your-redbrick-webspace.md:71]:
```suggestion
If you have any issues/questions drop a message in [`#helpdesk`](https://discord.redbrick.dcu.ie).
```

**Review by wizzdom — COMMENTED** [review]
Just adding links to discord

**Comment by Shockcoil131** [comment]
Im happy with it there now, thanks for picking up on the few little bits I missed

**Review by wizzdom — APPROVED** [review]
LGTM

**Pull request merged**